src/main/java/Duke.java

```diff
@@ -1,10 +1,108 @@
+import java.util.*;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
+    public static void main(String[] args) throws DukeException {
+        /*String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+         */
+        Scanner sc = new Scanner(System.in);
+        String Hello = "Hello! I'm Duke\nWhat can i do for you?";
+        System.out.println(Hello);
+
+        int flag = 0;
+        ArrayList<task> mylist = new ArrayList<>();
+
+        while(flag == 0) {
```

Comment:
Minor issue with spacing here between `while` and `(`: 
Should be `while (flag == 0) {` instead of `while(flag == 0) {`